src/components/selection_input/selection_input.ts

```diff
@@ -138,6 +142,12 @@ export class SelectionInput extends Component<Props, SpreadsheetChildEnv> {
       ) {
         this.store.updateColors(nextProps.colors || []);
       }
+      if (
+        nextProps.enabled?.join() !== this.props.enabled?.join() &&
+        nextProps.enabled?.join() !== this.store.enabled.join()
```

Comment:
deepEquals instead ?

Reply:
Got a stacktrace with deepEquals when running the tests :/